### PR TITLE
Isaicp 2216

### PR DESCRIPTION
**Pull request by idimopoulos**
By submitting a pull request to the Joinup repository:
- you accept the conditions in the Joinup Individual Contributor Assignment Agreement
- you accept the terms laid out in the the code of conduct.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         "drupal/console": "~0.10",
         "drupal/core": "~8.1.0",
         "drupal/facets" : "~8.1.0-alpha3",
-        "drupal/og": "dev-ISAICP-2362-DEV",
+        "drupal/og": "dev-update-group-mapping",
```
Comment (pfrenssen) [inline composer.json]: What is this? Where is the PR?

File: web/profiles/joinup/config/install/og.og_role.rdf_entity-collection-facilitator.yml [hunk 2]
```diff
@@ -16,21 +16,19 @@ permissions:
...
-  - 'view any unpublished news content'
   - 'create custom_page content'
   - 'edit any custom_page content'
   - 'delete any custom_page content'
   - 'edit any document content'
   - 'delete any document content'
   - 'edit any event content'
   - 'delete any event content'
-  - 'create rdf entity news'
+  - 'create rdf entity content'
```
Comment (pfrenssen) [inline web/profiles/joinup/config/install/og.og_role.rdf_entity-collection-facilitator.yml]: Why is this changed here? This ticket is about editing your own collections, nothing is mentioned about creating rdf entity content. I'm not even sure what that means, "rdf entity content" sounds very generic. Adding solutions to collections perhaps? This is out of scope for this ticket and there are no tests added for this change.

File: web/profiles/joinup/config/install/og.og_role.rdf_entity-collection-facilitator.yml [hunk 3]
```diff
@@ -16,21 +16,19 @@ permissions:
...
   - 'edit any news content'
   - 'delete any news content'
-  - 'update any navigation ogmenu_instance'
-  - 'add new links to og menu instance entities'
-  - 'create asset_distribution rdf_entity'
-  - 'create asset_release rdf_entity'
-  - 'create solution rdf_entity'
+  - 'create asset_distribution rdf entity'
+  - 'create asset_release rdf entity'
+  - 'propose solution rdf entity'
```
Comment (pfrenssen) [inline web/profiles/joinup/config/install/og.og_role.rdf_entity-collection-facilitator.yml]: `create solution rdf entity` is changed into `propose solution rdf entity`, but there is no code or tests in this PR that refer to this change. It is also not mentioned in the ticket that we need to change anything about solutions.

File: web/profiles/joinup/config/install/og.og_role.rdf_entity-collection-facilitator.yml [hunk 4]
```diff
@@ -16,21 +16,19 @@ permissions:
...
   - 'edit any event content'
   - 'delete any event content'
-  - 'create rdf entity news'
+  - 'create rdf entity content'
+  - 'view any unpublished news content'
   - 'create news content'
   - 'edit any news content'
   - 'delete any news content'
-  - 'update any navigation ogmenu_instance'
-  - 'add new links to og menu instance entities'
```
Comment (pfrenssen) [inline web/profiles/joinup/config/install/og.og_role.rdf_entity-collection-facilitator.yml]: Why are these permissions taken away from the facilitator? Are they not allowed to add OG menu items any more? This is out of scope for this ticket.

File: composer.json (hunk 1 above)
Comment (idimopoulos) [inline composer.json]: Hmm, you are right about the documentation. I should add the comment in the ticket as well.

I will answer this here and will also add a comment to the update and create a follow up to track this.
The pull request for the main reason of changing branches is https://github.com/amitaibu/og/issues/246.
The ticket is about the 'update group' permission. About the owner of the collection having a permission to edit the group. What I did, as we discussed, I took the latest version of the og and added a small fix for the update group permission until amitaibu responds to it and we fix it properly.

File: web/profiles/joinup/config/install/og.og_role.rdf_entity-collection-facilitator.yml (hunk 2 above)
Comment (idimopoulos) [inline web/profiles/joinup/config/install/og.og_role.rdf_entity-collection-facilitator.yml]: Reverted.
